packages/nodes-base/nodes/Netlify/DeployDescription.ts

```diff
@@ -0,0 +1,185 @@
+import { INodeProperties } from 'n8n-workflow';
+
+export const deployOperations = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		displayOptions: {
+			show: {
+				resource: [
+					'deploy',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Cancel',
+				value: 'cancelSiteDeploy',
+				description: 'Cancel a deploy'
+			},
+			{
+				name: 'Create',
+				value: 'createSiteDeploy',
+				description: 'Create a new deploy'
+			},
+			{
+				name: 'Get',
+				value: 'getSiteDeploy',
+				description: 'Returns the specified deploy',
+			},
+			{
+				name: 'Get All',
+				value: 'listSiteDeploys',
+				description: 'Returns all deploys of a site',
+			},
+			{
+				name: 'Rollback',
+				value: 'rollbackSiteDeploy',
+				description: 'Rollback site deploy'
+			}
+			// {
+			// 	name: 'Update',
+			// 	value: 'updateSiteDeploys',
+			// 	description: 'Updates a site deploy',
+			// },
+		],
+		default: 'listSiteDeploys',
+		description: 'The operation to perform.',
+	},
+] as INodeProperties[];
+
+export const deployFields = [
+	{
+		displayName: 'Site ID',
+		name: 'siteId',
+		required: true,
+		type: 'options',
+		typeOptions: {
+			loadOptionsMethod: 'getSites',
+		},
+		description: 'Enter the Site ID',
+		displayOptions:{
+			show: {
+				resource: [
+					'deploy'
+				],
+				operation: [
+					'getSiteDeploy',
+					'createSiteDeploy',
+					'listSiteDeploys',
+					'rollbackSiteDeploy'
+				],
+			}
+		}
+	},
+	{
+		displayName: 'Deploy ID',
+		name: 'deployId',
+		required: true,
+		type: 'string',
+		displayOptions:{
+			show: {
+				resource: [
+					'deploy'
+				],
+				operation: [
+					'getSiteDeploy',
+					'cancelSiteDeploy'
+				],
+			}
+		}
+	},
+		// ----- Get All Deploys ------ //
+	{
+		displayName: 'Return All',
+		name: 'returnAll',
+		type: 'boolean',
+		displayOptions: {
+			show: {
+				operation: [
+					'listSiteDeploys',
+				],
+				resource: [
+					'deploy',
+				],
+			},
+		},
+		default: false,
+		description: 'If all results should be returned or only up to a given limit.',
+	},
+	{
+		displayName: 'Limit',
+		name: 'limit',
+		type: 'number',
+		displayOptions: {
+			show: {
+				operation: [
+					'listSiteDeploys',
+				],
+				resource: [
+					'deploy',
+				],
+				returnAll: [
+					false,
+				],
+			},
+		},
+		typeOptions: {
+			minValue: 1,
+			maxValue: 200,
+		},
+		default: 50,
+		description: 'How many results to return.',
+	},
+	// ---- Create Site Deploy ---- //
```

Comment:
For consistency, we are using this style:
```suggestion
					// ----------------------------------
					//           resource: operation
					// ----------------------------------
```